README.md

```diff
@@ -36,12 +43,58 @@ $ docker build -t iojs-app
 $ docker run --rm -it iojs-app
 ```
 
-To run a single script, you can mount it in a volume under `/usr/src/app`. From the root of your application directory (assuming your script is named `index.js`):
+To run a single script, you can mount it in a volume under `/usr/src/app`. From
+the root of your application directory (assuming your script is named
+`index.js`):
 
 ```
-$ docker run -v ${PWD}:/usr/src/app -w /usr/src/app -it --rm iojs iojs index.js
+$ docker run -v ${PWD}:/usr/src/app -w /usr/src/app --it --rm iojs iojs index.js
```

Comment:
Thanks for noticing. I have opened docker-library/docs#216 to fix this upstream as well.